docs/manual-setup.adoc

```diff
@@ -20,7 +20,7 @@ Regardless you use `macos-setup.sh` or not, you should also install:
 
 == Node.js version notice
 
-Please use Node `v11.15.0` for everything but E2E test script.
+Please use Node `v12.18.3` for everything but E2E test script.
```

Comment:
Can we also align the automatic setup to that version? Especially, the `deployments/local-setup-environment/provisioning/install-nodejs.sh` script.